src/wasm-validator.h

```diff
@@ -109,6 +96,21 @@ struct WasmValidator : public WasmWalker {
   }
 
 private:
+
+  // the "in" label has a none type, since no one can receive its value. make sure no one breaks to it with a value.
+  struct ChildChecker : public WasmWalker {
```

Comment:
if we must move this for msvc, we'd need to rename it. Perhaps `LoopChildChecker`?